Makefile

```diff
@@ -34,7 +34,7 @@ test-template:
 	make -C ./app-service-template test
 
 test-sdk:
-	$(GO) test ./... -coverprofile=coverage.out ./...
+	$(GO) test ./... -coverprofile=coverage.out ./... -race
```

Comment:
This looks strange... with the double `./...` that was there before. In edgex-go it looks like this:

```
GOTESTFLAGS?=-race
.
.
.
GO111MODULE=on go test $(GOTESTFLAGS) -coverprofile=coverage.out ./...
```

Please use the edgex-go way of doing it.

Reply:
Also this should be added to the Makefile for the app-service-template.

Reply:
Ah I wondered what that was about @lenny-intel - should be good now thanks